scripts/exhafs_analysis.sh

```diff
@@ -469,9 +470,24 @@ ${NLN} ${COMINhafs_obs}/hafs.t${cyc}z.tldplr.tm00.bufr_d          tldplrbufr
 
 fi #USE_SELECT
 
+# Workflow will read from previous cycles for satbias predictors if ONLINE_SATBIAS is set to yes
+if [ ${ONLINE_SATBIAS} = "YES" ]; then
+  if [ ! -s ${COMhafsprior}/RESTART_analysis/satbias_hafs_out ] && [ ! -s ${COMhafsprior}/RESTART_analysis/satbias_hafs_pc.out ]; then
```

Comment:
The if statement requires both HAFS files not to exist before copying over GDAS bias correction files. If only one file exists, I think we would want the workflow to stop and throw an error, because something unexpected occurred. Would that happen?

Reply:
@jak5808 thanks!